nix/modules/nixos.nix

```diff
@@ -1,31 +0,0 @@
-{
-  lib,
-  pkgs,
-  config,
-  ...
-}:
-with lib; let
-  cli-cfg = config.programs.owmods-cli;
-  gui-cfg = config.programs.owmods-gui;
-in {
-  options.programs = {
-    owmods-cli = {
-      enable = mkEnableOption "owmods-cli program";
-    };
-    owmods-gui = {
-      enable = mkEnableOption "owmods-gui program";
-    };
-  };
-
-  config = mkIf (cli-cfg.enable || gui-cfg.enable) (
-    mkMerge [
-      {
-        environment.systemPackages = [
-          (mkIf cli-cfg.enable (pkgs.owmods-cli))
-          (mkIf gui-cfg.enable (pkgs.owmods-gui))
-          pkgs.mono
```

Comment:
> They aren't really needed, since the manager is self-contained and doesn't rely on other options/services.

Most of the manager is self contained however it *does* still require Mono installed along-side it and in the PATH. Although I agree and this should prolly be done at the package-level and not in a NixOS/HM module.

Reply:
Just added a wrapper that includes mono